.gitignore

```diff
@@ -1,2 +1,4 @@
 platforms/**/Cargo.lock
 target
+.idea/
```

Comment:
Speaking as an IntelliJ user: please edit your `.gitconfig` and add:

```
[core]
  excludesfile = /home/user/.gitignore
```

...and configure your editor-specific errata in there.

Otherwise every repo's `.gitignore` needs to be updated with every file that needs to be ignored by any editor anyone contributes to the project uses.